bin/reth/src/args/utils.rs

```diff
@@ -26,6 +29,10 @@ pub fn chain_spec_value_parser(s: &str) -> eyre::Result<Arc<ChainSpec>, eyre::Er
         "goerli" => GOERLI.clone(),
         "sepolia" => SEPOLIA.clone(),
         "dev" => DEV.clone(),
+        #[cfg(feature = "optimism")]
+        "base_goerli" => BASE_GOERLI.clone(),
```

Comment:
imo we should use kebab-case for chain names

Reply:
Good with me